shared/oae/api/oae.api.util.js

```diff
@@ -1866,7 +1866,5 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
      *
      * @return {String}   Event(s) that can fire when the content of an input field changes
      */
-    var getFormChangeEventNames = exports.getFormChangeEventNames = function() {
-        return $('html').hasClass('ie-lt10') ? 'change keyup paste cut' : 'change input';
-    };
+    var getFormChangeEventNames = exports.getFormChangeEventNames = return "change input";
```

Comment:
This is not valid syntax... An export needs to be a function, so this would have to be `var getFormChangeEventNames = exports.getFormChangeEventNames = function() {return "change input";}`. However, in this case we shouldn't need the function at all anymore since it's a IE9 workaround, so a better solution would be to take out the function completely and change all references to it with the string `'change input'`, like mentioned in the TODO comment.